microprofile/metrics/src/main/java/io/helidon/microprofile/metrics/MetricsCdiExtension.java

```diff
@@ -364,65 +440,55 @@ private void processInjectionPoints(@Observes ProcessInjectionPoint<?, ?> pip) {
     }
 
     /**
-     * Records the need to add a {@code SyntheticSimplyTimed} annotation to each JAX-RS endpoint method.
+     * Adds a {@code SyntheticSimplyTimed} annotation to each JAX-RS endpoint method.
      *
      * @param pat the {@code ProcessAnnotatedType} for the type containing the JAX-RS annotated methods
      */
     private void recordSimplyTimedForRestResources(@Observes
-                                                   @WithAnnotations({GET.class, PUT.class, POST.class, HEAD.class, OPTIONS.class,
-                                                                            DELETE.class, PATCH.class})
+                                                   @WithAnnotations({HttpMethod.class})
```

Comment:
Oh, interesting; I guess Jandex takes meta-annotations into account?

Reply:
I've switched back just in case there is some oddness that hasn't surfaced in the testing so far.